app/views/ht_registrations/show.html.erb

```diff
@@ -1,39 +1,20 @@
 <div class="col-sm-6">
   <h1><%= @registration.name %></h1>
+  
   <dl class="dl-horizontal">
-    <dt>Institution:</dt>
-    <dd><%= @registration.inst_link %></dd>
-    <dt>JIRA Ticket:</dt>
-    <dd><%= @registration.jira_link %></dd>
-    <dt>Contact Info:</dt>
-    <dd><%= @registration.contact_info %></dd>
-    <dt>Auth Rep Name:</dt>
-    <dd><%= @registration.auth_rep_name %></dd>
-    <dt>Auth Rep Email:</dt>
-    <dd>
-      <%= link_to(@registration.auth_rep_email, "mailto:#{@registration.auth_rep_email}") %>
-    </dd>
-    <dt>Auth Rep Date:</dt>
-    <dd><%= @registration.auth_rep_date %></dd>
-    <dt>DSP Name:</dt>
-    <dd><%= @registration.dsp_name %></dd>
-    <dt>DSP Email:</dt>
-    <dd>
-      <%= link_to(@registration.dsp_email, "mailto:#{@registration.dsp_email}") %>
-    </dd>
-    <dt>DSP Date:</dt>
-    <dd><%= @registration.dsp_date %></dd>
-    <dt>MFA Addendum:</dt>
-    <dd><%= @registration.mfa_addendum %></dd>
+    <% HTRegistrationPresenter::ALL_FIELDS.each do |field| %>
```

Comment:
My inclination would be to get `ALL_FIELDS` from the instance rather than binding to the specific class name here.